.ci/jenkins/Jenkinsfile

```diff
@@ -78,7 +78,7 @@ def RunToxTests(configurationMaps) {
     Closure runToxTestsMethod = { configMap ->
         UTILS.PropagateStatus("Prepare_environment", configMap["dockerContainerName"])
         sh """
-            NGRAPH_WHL=\$(docker exec ${configMap["dockerContainerName"]} find /home/ngraph/python/dist/ -name 'ngraph*.whl')
+            NGRAPH_WHL=\$(docker exec ${configMap["dockerContainerName"]} bash -c "find  /home/ngraph/python/dist/ -name 'ngraph*.whl' -printf '%Ts\t%p\n' | sort -nr | cut -f2 | head -n1")
```

Comment:
I made a simple test, where I copied current `*.whl` and changed its *version*  number part. In the result I get two files with identical size but different names. With such data above commands give incorrect results. Try `find /home/ngraph/python/dist/ -name 'ngraph*.whl' -print | sort -nr | head -n1` instead.

Reply:
Your line returns latest file sorting by name, while mine sorts by file creation date. You recieved incorrect result because the copied file had newer date.

I've run a test creating newer file and giving it lower version number and older file with higher version number. 
```
[lab_nerval ~]$ docker exec mchrusci bash -c "ls -al /home/ngraph/python/dist"
total 750576
drwxr-xr-x.  2 root root      4096 Sep 10 15:39 .
drwxr-xr-x. 11 root root      4096 Sep  8 08:39 ..
-rw-r--r--.  1 root root 256191765 Sep 10 15:37 ngraph-0.3.0-cp35-cp35m-linux_x86_64.whl
-rw-r--r--.  1 root root 256191765 Sep 10 14:09 ngraph-0.7.0-cp35-cp35m-linux_x86_64.whl
-rw-r--r--.  1 root root 256191765 Sep 10 15:36 ngraph-0.8.0-cp35-cp35m-linux_x86_64.whl
[lab_nerval ~]$ docker exec mchrusci bash -c "find  /home/ngraph/python/dist/ -name 'ngraph*.whl' -printf '%Ts\t%p\n' | sort -nr | cut -f2 | head -n1"
/home/ngraph/python/dist/ngraph-0.3.0-cp35-cp35m-linux_x86_64.whl
[lab_nerval ~]$ docker exec mchrusci bash -c "find /home/ngraph/python/dist/ -name 'ngraph*.whl' -print | sort -nr | head -n1"
/home/ngraph/python/dist/ngraph-0.8.0-cp35-cp35m-linux_x86_64.whl
```